opencti-platform/opencti-graphql/src/modules/organization/organization-domain.ts

```diff
@@ -134,4 +137,33 @@ export const childOrganizationsPaginated = async <T extends BasicStoreCommon> (c
   return listEntitiesThroughRelationsPaginated<T>(context, user, organizationId, RELATION_PART_OF, ENTITY_TYPE_IDENTITY_ORGANIZATION, true, args);
 };
 
+export const organizationDelete = async (context: AuthContext, user: AuthUser, organizationId: string) => {
+  const organization = await findById(context, user, organizationId);
+  if (!organization) {
+    throw AlreadyDeletedError({ organizationId });
```

Comment:
How can we know if it's an organization already deleted or an ID that just matches no organization? Maybe the nuance is not important

Reply:
I followed the same logic we have in internalDeleteElementById